ddtrace/tracer/tracer_test.go

```diff
@@ -493,7 +493,8 @@ func TestTracerPrioritySampler(t *testing.T) {
 	assert.EqualValues(s.context.samplingPriority(), s.Metrics[keySamplingPriority])
 	s.Finish()
 
-	tr.forceFlush() // obtain new rates
+	tr.flushChan <- nil
+	time.Sleep(100 * time.Millisecond)
```

Comment:
I'm not sure if you've seen this elsewhere in the repo, but if you did, we should factor it out. Sleeping to wait for asynchronous operations is a bit of a smell and a fluke waiting to happen. But I guess in this scenario we don't really have a better choice, so we can leave it.

Reply:
Yes, sleeping is gross, but I couldn't see a simpler way here.